### ISSUE TITLE
Destination S3: Make `s3_bucket_path` optional

### ISSUE THREAD

**Issue by markandrus** [airbytehq/airbyte#19763]
## Tell us about the problem you're trying to solve
When I create an S3 Destination, I don't want to specify an `s3_bucket_path` because, for my use case, this would introduce an unnecessary intermediate directory. Instead, I want to be able to pass an empty string or omit `s3_bucket_path` entirely and have things "just work".

Airbyte already puts the namespace and stream name in my S3 files' paths — I don't need the extra `s3_bucket_path` prefix.

## Describe the solution you’d like
In Airbyte's UI and API, I should be able to omit `s3_bucket_path`.

## Describe the alternative you’ve considered or used
By using Airbyte's API, I can actually create an S3 Destination with `s3_bucket_path` set to the empty string. But, the problem is, if I do this, I still don't get the desired path in S3. Instead, when syncing, Airbyte creates files in S3 at the following path:

```
s3://${BUCKET}//${NAMESPACE}/…
```

Note the double forward slashes (`//`). Instead, I want Airbyte to create

```
s3://${BUCKET}/${NAMESPACE}/…
```

In other words, it's not collapsing the forward slash when `s3_bucket_path` is the empty string.

Should this be a bug report? At the very least, should the API reject empty strings like the UI does?

## Additional context
N/A

## Are you willing to submit a PR?
Maybe?

**Comment by radmast** [comment]
Is there anything done on this front? This seems like a very important feature to have.

**Comment by octavia-squidington-iii** [comment]
At Airbyte, we seek to be clear about the project priorities and roadmap. This issue has not had any activity for 365 days, suggesting that it's not as critical as others. It's possible it has already been fixed. It is being marked as stale and will be closed in 20 days if there is no activity. To keep it open, please comment to let us know why it is important to you and if it is still reproducible on recent versions of Airbyte.
